tests/test_e2e/test_network.py

```diff
@@ -40,6 +47,14 @@ def test_ip_get(mycharm):
         },
     )
 
+    network = Network(
+        bind_addresses=[
+            BindAddress(
+                interface_name="",
+                addresses=[Address(hostname="", value="203.0.113.0", cidr="")],
+            ),
+        ],
+    )
```

Comment:
It seems like we should make use of the defaults here (even if just as a "good example"). And if `value` becomes required+positional, like my other comment, we'd have:

```python
network = Network(bind_addresses=BindAddress([Address("203.0.113.0")]))
```

Thoughts?